docker/Dockerfile-base

```diff
@@ -0,0 +1,31 @@
+# python 3
+FROM continuumio/anaconda3
```

Comment:
This file doesn't include the changes to the Dockerfile in 
https://github.com/e-mission/e-mission-server/pull/594

This is probably because you split the Dockerfile into two so the changes to `Dockerfile` were not merged into `Dockerfile-base`. Please merge the changes manually.

Also, as I pointed out in https://github.com/e-mission/e-mission-server/pull/617#issuecomment-443917798, after the change to miniconda, the server image is only 2GB. So I would recommend keeping one image for consistency but I will not oppose two images.